### DISCUSSION TITLE
Making "Date Created" a field

### DISCUSSION THREAD

**Discussion by gamoeri** [beetbox/beets#4432] [Q&A]
I'd like to organize my library based on "Date Added" to my personal collection, but since I've copied and pasted my library so many times in the process of retaining it, the "Date Created" field is much more accurate. "Date Added" is reset after adding it to a new library/software, while "Date Created" estimates when I obtained the file. 

However, I am unable to find this time field in the query-able field list (in "beet fields"). Is there another way to do this or could "Date Created" be added to this list? 

I love beets but this is a big setback for me and my music organization journey!

**Comment by sampsyo** [comment]
Hi! You can always add custom fields to the beets database if you like. Just use something like `beet modify created=<value> <query>` to attach a field named `created` to your items.

You might also want to use [the `types` plugin](https://beets.readthedocs.io/en/stable/plugins/types.html) to treat that field as a date instead of an arbitrary string.

**Reply by gamoeri** [reply]
Gotcha! This seems cool/promising. 

How would I access the "Date Created" section of the file so I don't have to add the custom field manually?

**Reply by sampsyo** [reply]
Aha, that involves mapping to a file _tag_. It depends on the actual file format you're using, but you will likely need to write a custom plugin to do so:
https://beets.readthedocs.io/en/stable/dev/plugins.html#extend-mediafile
